### DISCUSSION TITLE
Poderiam descrever as desvantagens do framework Bulma?

### DISCUSSION THREAD

**Discussion by leopatrocinio** [frontendbr/forum#1751] [Perguntas e Respostas]
Estou fazendo um artigo para faculdade, e estou encontrando pouca informação relacionada as desvantagens do framework bulma.

Thank you!

**Comment by glauberm** [comment]
É uma pergunta meio ampla, né. Estou usando em um projeto agora e uma coisa que me incomoda é o uso de css resets ao invés do normalize. Se você joga um `<h2>` no meio do markup, por exemplo, ele aparece completamente desestilizado e você tem que complementar com alguma outra classe ou estilos próprios. Mas essa é a proposta do framework e traz outras vantagens em contrapartida. Se a gente encarar isso como uma desvantagem, a ausência de implementação de JavaScript também seria uma? Porque isso por exemplo eu adoro e acho que é um dos motivos do crescimento atual do Bulma. Sei lá, acho complicado falar em desvantagens dessa maneira desacoplada. Desvantagens em relação a que?

**Comment by leopatrocinio** [comment]
Estou fazendo uma comparação entre Bootstrap e o Bulma, e estou cotando vantagens e desvantagens de ambosEm 4 de abr de 2020 23:41, Glauber Mota <notifications@github.com> escreveu:

É uma pergunta meio ampla, né. Estou usando em um projeto agora e uma coisa que me incomoda é o uso de css resets ao invés do normalize. Se você joga um <h2> no meio do markup, por exemplo, ele aparece completamente desestilizado e você tem que complementar com alguma outra classe ou estilos próprios. Mas essa é a proposta do framework e traz outras vantagens em contrapartida. Se a gente encarar isso como uma desvantagem, a ausência de implementação de JavaScript também seria uma? Porque isso por exemplo eu adoro e acho que é um dos motivos do crescimento atual do Bulma. Sei lá, acho complicado falar em desvantagens dessa maneira desacoplada. Desvantagens em relação a que?

**Comment by dionarodrigues** [comment]
Ao usar o Bulma em um recente projeto, levei em consideração o fato dele não possuir JS, deixando eu decidir como eu gostaria de implementar os scripts. Vejo que essa questão pode ser uma vantagem em determinados cenários.

Apesar dele não possuir um conjunto grande de componentes, como o Bootstrap por exemplo, ele possui diversas extensões. 

Outro ponto positivo está relacionado com a sua documentação e a maneira como o código foi escrito, sendo possível entender rapidamente como o sass foi desenvolvimento e como extender a mesma lógica ao criar novos componentes.

**Comment by klausmadeira** [comment]
"Desvantagens" que vejo no Bulma embora fazer layouts com ele é ótimo e rápido, é não ter de forma nativa itens como:  slideshow/banners/carrosel e "sub menu/dropdown/close" que no Bootstrap podem ser feitos facilmente. Sei que em [Creativebulma](https://creativebulma.net/) vendem esses itens. Enfim, na documentação citam que não é bem o foco depender muito de "JS" mas por outro lado considero básicos os itens que citei.

*Considerando v0.9.1
